docs/examples/academy/33. Analysing Imports Exports Data.ipynb

```diff
@@ -229,64 +228,62 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 7,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "total                       2431041659\n",
-       "MEG/AG                       866788361\n",
-       "Far East                     803025155\n",
-       "East Asia                    549184939\n",
-       "China (excl. HK & Macau)     422978179\n",
-       "North China                  250206821\n",
-       "West Africa                  225805648\n",
-       "South China                  172771358\n",
-       "Southeast Asia               154141100\n",
-       "USAC/USGC                    149115266\n",
+       "total                             2814003471\n",
+       "Persian Gulf                      1101131077\n",
+       "East Asia                          526131636\n",
+       "Russia Far East                    202566183\n",
+       "West Africa                        145545825\n",
+       "US Gulf Coast / Gulf of Mexico     139310077\n",
+       "South East Asia                    117397695\n",
+       "South America East Coast           113521997\n",
+       "East Australia and New Zealand      70636320\n",
+       "West Australia                      55952838\n",
        "dtype: int64"
       ]
      },
-     "execution_count": 6,
+     "execution_count": 7,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "imports_df.sum().sort_values(ascending=False).head(10)"
+    "imports_df.sum(numeric_only=True).sort_values(ascending=False).head(10)"
```

Comment:
This is the only meaningful change in this file - it fixes an issue where the `.sum` method was not working after a Pandas upgrade.